index.js

```diff
@@ -138,7 +138,12 @@ exports.Prismic.withContext = function(req, res, callback) {
 };
 
 exports.Prismic.preview = function(req, res) {
-  Prismic.withContext(req,res, function then(ctx) {
+  Prismic.withContext(req,res, function then(err, ctx) {
+    if (err.status == 404) {
+      res.status(404).send("404 not found");
+    } else {
```

Comment:
Shouldn't there be an if after the else? With this code it looks like we're going to send a 500 even when everything is good.